### ISSUE TITLE
Composite attributes require a # between them

### ISSUE THREAD

**Issue by begna112** [Dynobase/dynobase#165]
Is there a reason that composite attributes require a `#` between them? 

For example, if I'm making a database of workstations, I would like the `hostname` to just be the combination of the three other attributes.
```
${whid}${usecase}${hostnumber}
```

But this doesn't work unless I add `#` between each attribute.
```
${whid}#${usecase}#${hostnumber}
```

See below, where `sk` is the `#` string and the `hostname` is without. The intended output would be `AAA1COMP001`.
![image](https://user-images.githubusercontent.com/10567958/148822374-f501571f-38a5-458d-8707-3b42af591a20.png)

I fail to see the point of encapsulating attribute names in `${}` if they are useless without a separator.

**Comment by danbdo** [comment]
Yeah, I have that as well. Dup of https://github.com/Dynobase/dynobase/issues/151
